test/system/libs/helpers.bash

```diff
@@ -577,3 +577,86 @@ function is_fedora_rawhide() (
 
   return 0
 )
+
+
+# Creates a container with org.freedesktop.Flatpak.SessionHelper D-Bus interface
+#
+# Pulling of an image is taken care of by the function
+#
+# Parameters:
+# ===========
+# - container_name - name of the container
+function create_container_flatpak_dbus() (
+  local container_name
+
+  container_name="$1"
```

Comment:
Nitpick: these two lines can be combined.

I know there are some places in the code base where the lines are split, but it is an anti-pattern caused by a misunderstanding.  It's because ShellCheck [complains](https://www.shellcheck.net/wiki/SC2155) about cases where [command substitution](https://tldp.org/LDP/abs/html/commandsub.html) is used.